lib/validators/movie.js

```diff
@@ -2,7 +2,15 @@
 
 const Joi = require('joi');
 
-module.exports = Joi.object().keys({
+exports.payload = Joi.object().keys({
   title: Joi.string().min(1).max(255).required(),
   release_year: Joi.number().integer().min(1878).max(9999).optional()
 });
+
+exports.query = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
```

Comment:
When I add this, I get a request error saying that 'title or fuzzy_title is required' even though I label them as optional in exports.query...?